tests/test_dependency_review_warning_gate.py

```
@@ -0,0 +1,317 @@
+
+
+def test_run_gh_wraps_called_process_error() -> None:
+    command = ["gh", "api", "/repos/example/example/issues/1/comments"]
+    exc = subprocess.CalledProcessError(
+        returncode=1,
+        cmd=command,
+        output="out",
+        stderr="boom",
+    )
+
+    def _raise(*args, **kwargs):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_raise`

Add return type annotation: `Never`

(ANN202)